irods/connection.py

```diff
@@ -408,7 +446,11 @@ def read_file(self, desc, size=-1, buffer=None):
 
         return response.bs
 
-    def _login_native(self):
+    def _login_native(self, password=None):
```

Comment:
Added a password parameter in input, defaulted to None

The normal 'native' authentication is untouched (no parameter give = password is None = assigned to self.account.password as previously).

PAM will use this method by passing the temporary password provided by the irods server